app/src/main/java/com/example/justjet/Ui/LoginFragment.java

```diff
@@ -0,0 +1,125 @@
+package com.example.justjet.Ui;
+
+import android.app.Activity;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.NavController;
+import androidx.navigation.NavDestination;
+import androidx.navigation.Navigation;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.justjet.R;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+import java.util.Objects;
+
+
+public class LoginFragment extends Fragment implements View.OnClickListener {
+    public LoginFragment() {
+        // Required empty public constructor
+    }
+    FirebaseAuth firebaseAuth;
+    private FirebaseUser mUser;
+    private static EditText emailid, password;
+    private static Button loginButton;
+    private static TextView signUp;
+    private static LinearLayout loginLayout;
+
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                          @Nullable Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_login, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        final Bundle bundle = new Bundle();
+        bundle.putBoolean("test_boolean", true);
+
+        final NavController navController = Navigation.findNavController(requireActivity(), R.id.my_nav_host_fragment);
+        firebaseAuth = FirebaseAuth.getInstance();
+            mUser = FirebaseAuth.getInstance().getCurrentUser();
```

Comment:
@Riyabelle25 Currently the user needs to log in every time he/she opens the app. You can use `mUser` variable to skip the login screen if the user is already logged in. I would like you to implement this functionality. 
Keep in mind, when you would implement this, you would face another problem: When the user clicks back from the Home Fragment, the user will be navigated to the login fragment, and since the user is still logged in, it'll again be navigated to the Home Fragment. Just research on it and try to overcome this problem. I'm just giving some keywords as a hint. Google about `navOptions` and `setPopUpto()`.